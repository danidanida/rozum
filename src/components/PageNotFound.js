import React from "react";

const PageNotFound = () => {
  return <h2>Oups! Page not found.</h2>;
};

export default PageNotFound;
