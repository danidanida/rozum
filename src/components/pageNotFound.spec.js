import React from "react";
import PageNotFound from "./PageNotFound";
import renderer from "react-test-renderer";

describe("The 404 page", () => {
  it("renders a page", () => {
    const tree = renderer.create(<PageNotFound tags={["css", "html", "go"]} />);
    expect(tree).toMatchSnapshot();
  });
});
