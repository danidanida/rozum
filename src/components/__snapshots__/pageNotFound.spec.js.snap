// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The 404 page renders a page 1`] = `
<h2>
  Oups! Page not found.
</h2>
`;
